talk/basicconcepts/assert.tex

```diff
@@ -64,12 +64,12 @@
 \end{frame}
 
 \begin{frame}[fragile]
-  \frametitlecpp[17]{Static Assert}
+  \frametitlecpp[11]{Static Assert}
```

Comment:
I we had the 17 because of `std::is_floating_point_v`. I know it's confusing because we discuss `static_assert` :S

Reply:
Ah, indeed! I remember discussing that with you now. Still, the slide is about `static_assert`, not `std::is_floating_point_v`, so using C++17 gives the impression that `static_assert` itself is C++17 only. I think I will replace `_v` with `::value` then, what do you think?

Reply:
That's the general bummer with all those C++ version annotations on the slides. They currently mean what C++ version you need to compile the slide content.

I kind of also want to avoid writing `std::is_floating_point<T>::value`, because students should definitely use `std::is_floating_point_v<T>`. It's a dilemma.

Reply:
I think it's preferable to use `::value` so students don't get a wrong impression that `static_assert` is a C++17 feature. I got that impression the first time I saw this slide.

Reply:
ok, then go for it!